activities/tool-management/plausible-analytics.md

```diff
@@ -0,0 +1,23 @@
+---
+type: resource
+---
+
+# Plausible Analytics
+
+Plausible Analytics is a simple, lightweight (< 1 KB), open source and privacy-friendly analytics tool. It doesn’t use cookies.
+
+We're currently trialling Plausible (see [issue 64](https://github.com/publiccodenet/jekyll-theme/issues/64).)
+
+## Using Plausible Analytics
+
+Plausible Analytics is integrated into our [Jekyll-theme](https://github.com/publiccodenet/jekyll-theme), which is used by all our websites (since all are built with Github Pages). It gives us a simple open source analytics dashboard, including website metrics such as the number of unique visitors, the number of page views on individual pages and the referral sources that have sent traffic to our site.
+
+We will use this data to assess the performance of our site and to figure out what we can do to improve our future efforts.
+
+## Plausible management
+
+Plausible Analytics works by loading a Javascript script on the top navigation of our websites. The dashboard can be accessed by the Codebase Stewards and Communication Coordinator.
```

Comment:
```suggestion
Plausible Analytics works by loading a Javascript script on the top navigation of our websites. [Our Plausible dashboard is publicly shared]. It can be configured by the Codebase Stewards and Communication Coordinator.
```